src/assets/scripts/client/ui/SettingsController.js

```diff
@@ -177,20 +177,65 @@ export default class SettingsController {
      * Build the html for a select option.
      *
      * @for SettingsController
-     * @method _buildOptionTemplate
+     * @method _buildSelectOptionTemplate
      * @param optionData {array<string>}
      * @param selectedOption {string}
      * @return optionSelectTempalate {HTML Element}
      * @private
      */
-    _buildOptionSelectTemplate(optionData, selectedOption) {
+    _buildSelectOptionTemplate(optionData, selectedOption) {
         if (optionData.value === selectedOption) {
             return `<option value="${optionData.value}" selected>${optionData.displayLabel}</option>`;
         }
 
         return `<option value="${optionData.value}">${optionData.displayLabel}</option>`;
     }
 
+    /**
+     * Build text input form element
+     *
+     * @for SettingsController
+     * @method _buildTextInputTemplate
+     * @param option {object}
+     * @return $container {jquery Element}
+     */
+    _buildTextInputTemplate(option) {
+        const $container = $(UI_OPTION_CONTAINER_TEMPLATE);
+        const $label = $(UI_OPTION_LABEL_TEMPLATE);
+        const currentValue = GameController.game.option.getOptionByName(option.name);
+        const $selector = $(`<input class="form-input" step="any"
```

Comment:
For larger bits of HTML, I _think_ we put them in string templates at the top of the file. Check me on that, cause it's been a while since I've been in the code here. I'm ok with it here, but I'd like to keep things consistent if we can.

Reply:
For larger blocks (multiple HTML elements), you're correct. However, this is just a single element with several attributes so it looks longer than it really is.

Reply:
Works for me, thanks for checking!